hack/build-and-push.sh

```diff
@@ -64,10 +64,9 @@ fi
 
 # First, build the image, with the version info passed in.
 # Note that an image will *always* be built tagged with the GIT_TAG, so we know when it was built.
-docker build --build-arg COMMIT=${COMMIT} --build-arg TAG=${BINARY_TAG} \
-  			-t ${REGISTRY}/admission-server:${GIT_TAG} .
-docker push ${REGISTRY}/admission-server:${GIT_TAG}
+docker buildx build --build-arg COMMIT=${COMMIT} --build-arg TAG=${BINARY_TAG} \
+  			-t ${REGISTRY}/admission-server:${GIT_TAG} . --platform linux/amd64,linux/arm64 --push
 
 # Then, we add an extra version tag - either :latest or semver.
-docker tag ${REGISTRY}/admission-server:${GIT_TAG} ${REGISTRY}/admission-server:${VERSION_TAG}
-docker push ${REGISTRY}/admission-server:${VERSION_TAG}
+docker buildx build --build-arg COMMIT=${COMMIT} --build-arg TAG=${BINARY_TAG} \
+  			-t ${REGISTRY}/admission-server:${VERSION_TAG} . --platform linux/amd64,linux/arm64 --push
```

Comment:
If I'm not mistaken, we shouldn't have needed to change the tag line at all now that we're building a multi-arch image? Am I mistaken?

Reply:
`buildx build` can do build+push multiple tags in one command.

This should be one `docker` exec with two `-t` flags passed